imports/schemas/userEvent.coffee

```diff
@@ -4,22 +4,32 @@ userEventSchema = new SimpleSchema(
     optional: true
   articleCount:
     type: Number
+    optional: true
```

Comment:
[I believe that](https://github.com/aldeed/meteor-simple-schema#optional) `optional: true` is the default in simple schema, are all of these required?